examples/web-components-kitchen-sink/custom-elements.json

```diff
@@ -4,28 +4,51 @@
     {
       "name": "demo-wc-card",
       "description": "This is a container looking like a card with a back and front side you can switch",
-      "properties": [
+      "jsDoc": "/**\n * This is a container looking like a card with a back and front side you can switch\n *\n * @slot - This is an unnamed slot (the default slot)\n * @fires side-changed - Fires whenever it switches between front/back\n * @cssprop --demo-wc-card-header-font-size - Header font size\n * @cssprop --demo-wc-card-front-color - Font color for front\n * @cssprop --demo-wc-card-back-color - Font color for back\n */",
+      "attributes": [
+        {
+          "name": "back-side",
+          "description": "Indicates that the back of the card is shown",
+          "jsDoc": "/**\n     * Indicates that the back of the card is shown\n     */",
+          "type": "boolean"
+        },
         {
           "name": "header",
-          "type": "String",
-          "attribute": "header",
-          "description": "Shown at the top of the card",
-          "default": "Your Message"
+          "description": "Header message",
+          "jsDoc": "/**\n     * Header message\n     */",
+          "type": "string"
         },
         {
           "name": "rows",
-          "type": "Array",
-          "attribute": "rows",
-          "description": "Tabular data shown on the back of the card",
-          "default": []
+          "description": "Data rows",
+          "jsDoc": "/**\n     * Data rows\n     */",
+          "type": "never[]"
+        }
+      ],
+      "properties": [
+        {
+          "name": "side",
```

Comment:
we should get rid of this by removing the setter/getter for `side` 🤗

Reply:
I left this in there because it triggers the `side-changed` event which is documented by the `@fires side-changed`. This gives an example of all the things the `custom-elements.json` file can provide. 

I can still remove it though if you want.

Reply:
ah right... for the event let's better do this 🤗 
```js
  set backSide(value) {
    this.__backSide = value;
    this.dispatchEvent(new Event('side-changed'));
    this.requestUpdate();
  }

  get backSide() {
    return this.__backSide;
  }
```

sorry for the many change request 🙈

Reply:
dam this "disables" the attribute sync...

Reply:
that works
```js
  _requestUpdate(name, oldValue) {
    // @ts-ignore
    super._requestUpdate(name, oldValue);

    if (name === 'backSide') {
      this.dispatchEvent(new Event('side-changed'));
    }
  }
```

but requires the private api `_requestUpdate`... 

so I'm not sure what is best in that case... feel free to choose 👍

Reply:
I think I have a solution, also fixed some css issues.

Reply:
awesome 👍